pkgs/development/python-modules/pylibftdi/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, httpserver
+, libftdi1
+, libusb1
+}:
+
+buildPythonPackage rec {
+  pname = "pylibftdi";
+  version = "0.17.0";
+  disabled = pythonOlder "3"; # compatibility with httpserver
```

Comment:
This line is probably not needed if you bump to 0.18.1, which claims to support python 2.

Reply:
I did this because httpserver complains about python2. Is it better to remove it and let httpserver trigger the error?

Reply:
Yes, sure